modules/affile/affile-source.c

```diff
@@ -253,6 +253,43 @@ affile_sd_construct_proto(AFFileSourceDriver *self, gint fd)
     }
 }
 
+static void
+affile_sd_reset_file_state(PersistState *state, const gchar *old_persist_name)
+{
+  gchar *new_persist_name = g_strdup_printf("%s_DELETED",old_persist_name);
+  persist_state_rename_entry(state,old_persist_name,new_persist_name);
+  g_free(new_persist_name);
+}
+
+static inline gboolean
+_setting_logpipe(LogPipe *s)
```

Comment:
please let the compiler decide whether inline makes sense or not. (e.g. drop that qualifier).